tests/test_cli.py

```diff
@@ -38,6 +39,24 @@ def test_list_projects_bad_arguments():
         annif.cli.run_list_projects, ['moi', '--debug', 'y']).exit_code != 0
 
 
+def test_list_projects_config_path_option():
+    nonexistent_file = "nonexistent.cfg"
+    result = runner.invoke(
+        annif.cli.cli, ["list-projects", "--projects", nonexistent_file])
+    assert not result.exception
+    assert result.exit_code == 0
+    assert 'Project configuration file "{}" is missing.'.format(
+        nonexistent_file) in result.output
+
+    result = runner.invoke(
```

Comment:
This is testing a different situation (existing config file), so it should be separated into its own test.